src/index.html

```diff
@@ -38,6 +38,79 @@
     <!-- End Header -->
 
     <!-- Start Page Content -->
+    <section
+      class="offerings-section items-center justify-center flex flex-col relative bg-color-pink py6"
+    >
+      <div class="absolute t0 w100">
+        <img class="w100" src="img/pink-background-shape-1.svg" alt="Wavy shape" />
+      </div>
+
+      <div class="pink-background-bottom-shape absolute w100">
+        <img class="w100" src="img/pink-background-shape-2.svg" alt="Wavy shape" />
```

Comment:
These kinds of decorative images should have role="presentation" and empty alt tag  https://www.w3.org/WAI/tutorials/images/decorative/#image-used-as-part-of-page-design

Reply:
Oooh nice thank you!! Hehe "wavy shape"